ci-operator/config/rh-ecosystem-edge/kernel-module-management/rh-ecosystem-edge-kernel-module-management-main.yaml

```diff
@@ -0,0 +1,32 @@
+binary_build_commands: make manager
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+images:
+  - dockerfile_literal: |
+      FROM src
+      RUN make golangci-lint
+    from: src
+    to: linter
```

Comment:
Why are we building this image? Can't we just use `src`? We are testing the lint job anyway.

Reply:
This was an attempt at installing golangci-lint before running the `make` target. I will revert it.